Update, add and drop Python versions.

jugmac00 opened the pull request:
- update PyPy version
cf. https://docs.travis-ci.com/user/languages/python/#pypy-support
- add support for Python 3.7
- drop support for Python 3.4

jugmac00 committed:
Update PyPy version for Travis.

cf. https://docs.travis-ci.com/user/languages/python/#pypy-support

modified:   .travis.yml
modified:   CHANGES.rst

jugmac00 committed:
Add support for Python 3.7.

This closes #3.

modified:   .travis.yml
modified:   CHANGES.rst
modified:   setup.py
modified:   tox.ini

jugmac00 committed:
Drop support for Python 3.4.

modified:   .travis.yml
modified:   CHANGES.rst
modified:   setup.py
modified:   tox.ini

Commit:
Enable coverage computation.

Commit:
Configure coverage.

icemac reviewed (approved):
LGTM.

It would be nice to additionally add support for Python 3.8.

jugmac00 committed:
Add support for Python 3.8.

modified:   .travis.yml
modified:   CHANGES.rst
modified:   setup.py
modified:   tox.ini
